visualization/app/codeCharta/ui/codeMap/codeMap.po.ts

```diff
@@ -1,6 +1,13 @@
 export class CodeMapPageObject {
 	async rightClickMap() {
 		await expect(page).toClick("#codeMap", { button: "right", timeout: 3000 })
-		await page.waitForSelector("node-context-menu-component", { visible: false })
+	}
+
+	// TODO: activate this function call
+	//  if the new puppeteer mouse wheel API is supported by it's type definition:
+	// https://github.com/DefinitelyTyped/DefinitelyTyped/issues/48710
```

Comment:
Please already use the API and ignore the type error with a comment as in `// @ts-ignore`.